src/components/chart/buggy-basic-chart.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import _ from 'underscore'
+import './chart.css';
+import Cardiogram from '../../models/cardiogram';
+
+const REFRESH_INTERVAL_MS = 30 * 1000;  // refresh every 30 seconds
+
+/**
+ *  A React component which displays a heart rate bar chart, with time on the X axis and heart rate
+ *  on the Y axis.
+ *
+ *  TODO(brandon): fix all these darn bugs!
+ */
+export default class BuggyHeartRateChart extends React.Component {
```

Comment:
Candidate may point out that we should use a functional component or `extend React.PureComponent` since we don't mutate the state/props